docs/deploy/agent-engine/deploy.md

```diff
@@ -0,0 +1,188 @@
+
+    ```shell
+    gcloud auth application-default login
+    ```
+
+    You should now see "You are now authenticated with the gcloud CLI!".
+
+*   (Optional) If you need to set or change your default project in gcloud, you
+    can use:
+
+    ```shell
+    gcloud config set project MY-PROJECT-ID
+    ```
+
+### Define your agent {#define-your-agent}
+
+With your Google Cloud and coding environment prepared, you're ready to deploy
+your agent. The instructions assume that you have an agent project folder,
+such as:
+
+```shell
+multi_tool_agent/
+├── .env
+├── __init__.py
+└── agent.py
+```
+
+You can also refer to the sample agent on Github: [multi_tool_agent](https://github.com/google/adk-docs/tree/main/examples/python/snippets/get-started/multi_tool_agent).
+
+## Deploy the agent {#deploy-agent}
+
+You can deploy from your terminal using the `adk deploy` command line tool. This
+process packages your code, builds it into a container, and deploys it to the
+managed Agent Engine service. This process can take several minutes.
+
+### adk deploy agent_engine
+
+The following example deploy command uses the `multi_tool_agent` sample code as
+the project to be deployed:
+
+```shell
+PROJECT_ID=my-project-id
+LOCATION=us-central1
+GCS_BUCKET=gs://MY-CLOUD-STORAGE-BUCKET
+
+adk deploy agent_engine \
+        --project=$PROJECT_ID \
+        --region=$LOCATION \
+        --staging_bucket=$GCS_BUCKET \
+        --display_name="My First Agent" \
+        multi_tool_agent
+```
+
+For `region`, you can find a list of the supported regions on the
+[Vertex AI Agent Builder locations page](https://docs.cloud.google.com/agent-builder/locations#supported-regions-agent-engine).
+To learn about the CLI options for the `adk deploy agent_engine` command, see the
+[ADK CLI Reference](https://google.github.io/adk-docs/api-reference/cli/cli.html#adk-deploy-agent-engine).
+
+### Deploy command output
+
+Once successfully deployed, you should see the following output:
+
+```shell
+Creating AgentEngine
+Create AgentEngine backing LRO: projects/123456789/locations/us-central1/reasoningEngines/751619551677906944/operations/2356952072064073728
+View progress and logs at https://console.cloud.google.com/logs/query?project=hopeful-sunset-478017-q0
+AgentEngine created. Resource name: projects/123456789/locations/us-central1/reasoningEngines/751619551677906944
+To use this AgentEngine in another session:
+agent_engine = vertexai.agent_engines.get('projects/123456789/locations/us-central1/reasoningEngines/751619551677906944')
+Cleaning up the temp folder: /var/folders/k5/pv70z5m92s30k0n7hfkxszfr00mz24/T/agent_engine_deploy_src/20251219_134245
+```
+
+Note that you now have a `RESOURCE_ID` where your agent has been deployed (which
+in the example above is `751619551677906944`). You need this ID number along
+with the other values to use your agent on Agent Engine.
+
+## Using an agent on Agent Engine
+
+To use your agent on Agent Engine, you need the following:
```

Comment:
Suggest adding an introductory sentence to this section to mention that you can query the agent using the Vertex AI SDK, Python requests library, or a REST API client. Otherwise the section is sparse and not clear on what the user action should be.

And consider adding a link to the "Test deployed agent page" that shows two of these methods.